backend/server.go

```
@@ -15,11 +15,20 @@ func NewServer(address string) *Server {
 }
 
 func (s *Server) Start() error {
+	http.HandleFunc("/health", func(w http.ResponseWriter, r *http.Request) {
+		if r.Method != http.MethodGet {
+			w.WriteHeader(http.StatusMethodNotAllowed)
+			return
+		}
+
+		w.Header().Set("Content-Type", "application/json")
+		w.WriteHeader(http.StatusOK)
+		w.Write([]byte(`{"status":"healthy"}`))
```

golangci-lint (1.62.2)

Error return value of `w.Write` is not checked

(errcheck)